components/SessionInitiation.js

```diff
@@ -55,39 +55,45 @@ export default class SessionInitiation extends React.Component {
     return fetchedArtistIds.concat(topArtistIds, followedArtistIds, libraryArtistIds);
   }
 
-  updateUserSeeds = async (access_token, refresh_token, artist_ids) => {
-    let updated = await this.props.updateUserSeeds(access_token, refresh_token, artist_ids);
-    if (updated) {
-      await this.props.syncedNewUser(access_token, refresh_token);
-    } else {
-      // did not update user seeds
-    }
+  syncUserWithSpotify = async (access_token, refresh_token, artist_ids) => {
+    let updated = await this.props.syncUserWithSpotify(access_token, refresh_token, artist_ids);
+    return updated;
```

Comment:
`return await this.props.syncUserWithSpotify(access_token, refresh_token, artist_ids);`